bdc_motor/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "0.1.0"
```

Comment:
If you don't expect any changes in foreseeable future, I'd suggest to start with 1.0.0. But it is up to you

Reply:
IMO, the APIs there are close to stable, but at the moment, this is still a very new library, better to give the users some time to test it and feedback.